helm/charts/determined/values.yaml

```diff
@@ -81,6 +81,20 @@ checkpointStorage:
   # secretKey: <secret_key>
   # endpointUrl: <endpoint_url>
 
+  # The following checkpoint configurations are mutually exclusive.
+  # Please only specify one to use Azure Blob Storage.
```

Comment:
hmm - I think it might be repetitive to have two warnings - would you mind illustrating what you mean by having warnings for each item?

Reply:
Yeah so instead of having lines 84 & 85, you could have line 87 look like:

`For storing in Azure Blob Storage with a connection string. Do NOT use if already using Blob Storage with account URL`

Similar for line 92.

Reply:
Sounds good - will do!